chapter_convolutional-modern/batch-norm_vn.md

```diff
@@ -116,7 +116,10 @@ You might think that this noisiness should be a problem.
 As it turns out, this is actually beneficial.
 -->
 
-*dịch đoạn phía trên*
+Lưu ý rằng chúng ta thêm một hằng số $\epsilon > 0$ vào biểu thức tính phương sai để đảm bảo không thực hiện phép chia cho không khi chuẩn hoá, ngay cả trong trường hợp giá trị ước lượng phương sai thực nghiệm bằng không.
+$\hat{\mathbf{\mu}}_\mathcal{B}$ và $\hat{\mathbf{\sigma}}_\mathcal{B}$ giải quyết vấn đề chênh lệch tỷ lệ bằng việc sử dụng của giá trị trung bình và phương sai có nhiễu.
```

Comment:
```suggestion
Các ước lượng $\hat{\mathbf{\mu}}_\mathcal{B}$ và $\hat{\mathbf{\sigma}}_\mathcal{B}$ đương đầu với những vấn đề của các mạng sâu bằng việc sử dụng giá trị trung bình và phương sai có nhiễu.
```
scaling ở đây mình hiểu là scale NN lên thành nhiều tâng sâu hơn ấy.

Reply:
@lkhphuc , dạ em chưa hiểu chỗ scale NN lên thành nhiều tầng sâu hơn là có ý gì ạ. Cái này theo em đọc, sau khi họ chuẩn hoá các giá trị theo các số thông kê của minibatch, thì có thể làm mất đi những tính chất gì mà tầng đó đại diện khi đi qua hàm kích hoạt.  Ví dụ như mình chuẩn hoá xong thì giá trị của nó chỉ nằm trong phân phối (0,1). Khi đi qua một hàm nonlinear activation, ví dụ như là sigmoid thì mình trong khoảng (-1, 1) này thì sigmoid nó chỉ còn là xấp xỉ với hàm linear, từ khoảng giá trị đầu vào từ 2 đến 4 thì nó mới thể hiện được tính non-linear. Vì vậy họ thêm hai tham số gamma với beta để cho nó đảm bảo là khi mà mình biến đổi thì vẫn giữ được những tính chất mà tầng đại diện (identity transformation).

Reply:
à ko bạn hiểu đúng rồi. lúc đó mình đọc nhầm :)).

Reply:
hm mình có chút vướng mắc. Phía dưới đoạn `counteract the scaling issue by using noisy estimates` có đề cập đến cái bị `noise` `counteract`:
> For reasons that are not yet well-characterized theoretically, various sources of noise in optimization often lead to faster training and less overfitting.

Có nghĩa `the scaling issue` là `slow training and much overfitting`, tức ứng với vấn đề được đề cập ở đầu phần này (dòng 44 trong file):
> 3. Deeper networks are complex and easily capable of overfitting.

tức là `the scaling issue` là issue khi build deep network, thêm nhiều tầng hơn.
Mn nghĩ sao?

Reply:
@lkhphuc  @cuongvng  em bị nhầm, đọc mà không để ý chữ noisy, 2 anh đúng rồi ạ. Đọc cả 2 đoạn thì nó đang đề cập đến tác dụng của nhiễu trong ngữ cảnh giúp huấn luyện nhanh hơn và bớt quá khớp đi.

Reply:
haha sáng tối mỗi lúc đọc hiểu một ý.
Vậy thống nhất sửa như mình gợi ý ở trên cho giống ý deep net nhé.

Reply:
Em đề xuất thế này

```suggestion
Các ước lượng $\hat{\mathbf{\mu}}_\mathcal{B}$ và $\hat{\mathbf{\sigma}}_\mathcal{B}$ giúp đương đầu với vấn đề khi cần mở rộng số tầng của mạng (mạng học sâu hơn) bằng việc sử dụng nhiễu của ước tính giá trị trung bình và phương sai.
```